source/computer/macos-setup.rst

```diff
@@ -1,9 +1,171 @@
 macOS 配置指南
 ==============
 
+:本节作者: `王亮 <https://github.com/wangliang1989>`__\（作者）、
+           `田冬冬 <https://me.seisman.info/>`__\（作者）、
+           `姚家园 <https://github.com/core-man>`__\ （审稿）
+:最后更新日期: 2021-01-11
+
+----
+
+.. warning::
+
+   本文成稿于 2018 年 1 月，并于 2021 年 1 月做少量改动。
+   但笔者尚未在新电脑上进行测试，无法保证所有命令均正确。欢迎反馈。
+
+第一次启动 Mac 电脑后，经过简单的设置，就得到了一个可供日常使用的 macOS 系统，
+但这样的 macOS 系统尚不能满足日常科研与编程开发的需求，还需要做进一步的配置。
```

Comment:
I don't remember the details, but perhaps we should mention that the account name should be in English and no whitespace.

Reply:
You mean our mac account?

Reply:
Yes.

Reply:
Like `/Users/seisman`.

Reply:
I asked apple people today in APP Store. He said we can use Chinse if we set the language to Chinese.

Reply:
As you just had a fresh install yesterday, you may know it better. Did you need to set up an account when starting your Mac for the first time? What about `root` user? Did you set the password?